client/dashboard/sites/settings-repositories/connect-repository.tsx

```diff
@@ -49,17 +49,17 @@ export default function ConnectRepository() {
 				<PageHeader
 					prefix={ <Breadcrumbs length={ 3 } /> }
 					title={ __( 'Connect Repository' ) }
-					description={ __( 'Connect a GitHub repository to deploy code to your WordPress site.' ) }
+					description={ __( 'Deploy code from GitHub to your WordPress site.' ) }
```

Comment:
Maybe we could consider using `WordPress.com` instead `WordPress`:

```suggestion
					description={ __( 'Deploy code from GitHub to your WordPress.com site.' ) }
```

But generally I prefer shorter text.